app/services/appointment_notification_service.rb

```diff
@@ -19,20 +19,20 @@ def send_after_missed_visit
     next_messaging_time = Communication.next_messaging_time
 
     eligible_appointments.each do |appointment|
-      appointment_reminder = create_appointment_reminder(appointment)
-      AppointmentNotification::Worker.perform_at(next_messaging_time, appointment_reminder.id)
+      notification = create_appointment_reminder(appointment)
+      AppointmentNotification::Worker.perform_at(next_messaging_time, notification.id)
     end
   end
 
   private
 
   def create_appointment_reminder(appointment)
-    AppointmentReminder.create!(
-      appointment: appointment,
+    Notification.create!(
+      subject: appointment,
       patient: appointment.patient,
       remind_on: appointment.remind_on,
       status: "scheduled",
-      message: "sms.appointment_reminders.#{communication_type}"
```

Comment:
I still think the locale keys are probably the only thing we should leave as is.